lib/msal-common/src/utils/Constants.ts

```diff
@@ -96,7 +96,10 @@ export enum AADServerParamKeys {
     X_CLIENT_OS = "x-client-OS",
     X_CLIENT_CPU = "x-client-CPU",
     POST_LOGOUT_URI = "post_logout_redirect_uri",
-    DEVICE_CODE = "device_code"
+    DEVICE_CODE = "device_code",
```

Comment:
Should we create a separate Enum? Can be done separately and not just these but the general constants lumped together - should they be divided?